aspnetcore/host-and-deploy/visual-studio-publish-profiles.md

```diff
@@ -340,7 +332,15 @@ If you require *web.config* transformations (for example, setting environment va
 
 ## Exclude files
 
-When publishing ASP.NET Core web apps, the build artifacts and contents of the *wwwroot* folder are included. `msbuild` supports [globbing patterns](https://gruntjs.com/configuring-tasks#globbing-patterns). For example, the following `<Content>` element excludes all text (*.txt*) files from the *wwwroot/content* folder and all its subfolders.
+When publishing ASP.NET Core web apps, the following assets are included:
+
+* Build artifacts
+* Folders and files matching the following globbing patterns:
+  * `**\*.config` (for example, *web.config*)
+  * `**\*.json` (for example, *appsettings.json*)
+  * `wwwroot\**`
+
+`msbuild` supports [globbing patterns](https://gruntjs.com/configuring-tasks#globbing-patterns). For example, the following `<Content>` element excludes all text (*.txt*) files from the *wwwroot/content* folder and its subfolders.
```

Comment:
Is "\`msbuild\`" supposed to be code fenced? Shouldn't it just be "MSBuild"?

I also used a backslash in the suggestion ... looks like that's the common pattern in the topic.

```suggestion
MSBuild supports [globbing patterns](https://gruntjs.com/configuring-tasks#globbing-patterns). For example, the following `<Content>` element excludes all text (*.txt*) files from the *wwwroot\content* folder and its subfolders.
```